package-list.json

```diff
@@ -13,14 +13,23 @@
       "fhirversion": "4.0.1",
       "current": true
     },
+    {
+      "version": "0.3.0-ballot",
+      "date": "2024-03-10",
+      "desc": "This snapshot is published for ballot and the March 2024 Connectathon.",
```

Comment:
The desc is not sufficient, but since it matches what is already in the history it must be kept.

Going forwards enough information to know what the snapshot is for, e.g. Ballot for Comment etc. needs to be provided. Preferably 2-3 sentences.

You may resolve this conversation, but ensure that the testing snapshot is released with a meaningful desc.

Reply:
noted